### PR TITLE
fix(comparison): calculate statistics for multiple layers

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
It is not intuitive to show just url in **Layer(s) used** section. More than that it is difficult to show such info for BSLs or for WebScenes.
New logic of **Layer(s) used** stattistics:
- If loaded single layer we show its name
- If loaded BSL layer we show BSL name 
- If loaded WebScene we show all active webscene layers names
Also you can copy url by copy button as before.
Example of single layer:
image
Example of BSL:
image
Example of WebScene:
image

### CODE REVIEW

File: src/components/comparison/memory-usage-panel/memory-usage-panel.tsx [hunk 1]
```diff
@@ -126,19 +128,26 @@ export const MemoryUsagePanel = ({
...
-                    style={{ cursor: "pointer" }}
-                    onClick={() => {
-                      navigator.clipboard.writeText(tilesetStats.id);
-                    }}
-                  />
-                </StatContainer>
+                {
+                  tilesetStats.id.split('<-tileset->').map(tilesetUrl => {
+                    const activeLayer = activeLayers.find(layer => layer.url === tilesetUrl);
+                    const activeLayerName = activeLayer?.name || `${tilesetUrl.substring(0, 37)}...`;
```
Comment (belom88) [inline src/components/comparison/memory-usage-panel/memory-usage-panel.tsx:134]: what the magic number 37? If it is to fit to the panel width, use `text-overflow: ellipsis` css property

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/components/comparison/comparison-side/comparison-side.tsx [hunk 2]
```diff
@@ -619,7 +657,7 @@ export const ComparisonSide = ({
         onWebGLInitialized={onWebGLInitialized}
         onTilesetLoad={(tileset: Tileset3D) => onTilesetLoadHandler(tileset)}
         onTileLoad={onTileLoad}
-        onAfterRender={onAfterDeckGlRender}
+        onAfterRender={handleOnAfterRender}
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply src/components/comparison/comparison-side/comparison-side.tsx:697]: Viewer-app is also using `onAfterRender` hook. Statistics isn't shown without it.

File: src/components/comparison/memory-usage-panel/memory-usage-panel.tsx [hunk 3]
```diff
@@ -126,19 +128,26 @@ export const MemoryUsagePanel = ({
...
+                    const isEmptyName = activeLayerName === '...';
+
+                    return !isEmptyName ? (
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply src/components/comparison/memory-usage-panel/memory-usage-panel.tsx]: Done

File: src/components/comparison/comparison-side/comparison-side.tsx [hunk 4]
```diff
@@ -322,6 +324,37 @@ export const ComparisonSide = ({
...
+    const activeLayersCount = activeLayers.length;
+
+    if (loadedTilesetsCount && activeLayersCount) {
+      const isBSL = loadedTilesetsCount > activeLayersCount;
+      const statsName = isBSL ?
+        // Use Building Scene Url for stats id.
+        activeLayers[0].url
+        :
+        // Use loaded tilesets urls to create stats id.
+        loadedTilesets.map(loadedTileset => loadedTileset.url).join('<-tileset->');
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply src/components/comparison/comparison-side/comparison-side.tsx]: Done

File: src/components/comparison/memory-usage-panel/memory-usage-panel.tsx (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/components/comparison/memory-usage-panel/memory-usage-panel.tsx]: Done

**Review by belom88 — APPROVED** [review]

**Pull request merged**